File: javascript/par_impar.js
const n = parseInt(prompt("Quantos numeros voce vai digitar?"))

let valor;

for (i = 0; i < n; i++) {
    valor = parseInt(prompt("Digite um numero:"));

    if (valor > 0) {
        if (valor % 2 == 0) {
            console.log("PAR POSITIVO");
        } else {
            console.log("IMPAR POSITIVO");
        }
    } else if(valor < 0) {
        if (valor % 2 == 0) {
            console.log("PAR NEGATIVO");
        } else {
            console.log("IMPAR NEGATIVO");
        }
    } else {
        console.log("NULO");
    }
}

File: python/par_impar.py
n = int(input("Quantos numeros voce vai digitar? "))

for i in range(0, n):
    valor = int(input("Digite um numero: "))

    if valor > 0:
        if valor % 2 == 0:
            print("PAR POSITIVO")
        else:
            print("IMPAR POSITIVO")
    elif valor < 0:
        if valor % 2 == 0:
            print("PAR NEGATIVO")
        else:
            print("IMPAR NEGATIVO")
    else:
        print("NULO")
